tron-protocol/src/main/scala/org/tron/core/UTXOSet.scala

```diff
@@ -33,7 +33,7 @@ class UTXOSet(
       val len = txOutputs.outputs.size
       for (i <- 0 until len) {
         val txOutput = txOutputs.outputs(i)
-        if (pubKey.hex == ByteArray.toHexString(txOutput.pubKeyHash.toByteArray) && accumulated < amount) {
+        if (pubKey.addressHex == ByteArray.toHexString(txOutput.pubKeyHash.toByteArray) && accumulated < amount) {
```

Comment:
I think switching these evaluations would be slightly better.

So `accummulated < amount && (pubKey.addressHex == ByteArray.toHexString(txOutput.pubKeyHash.toByteArray)`

Furthermore, I think that `accumulated < amount` can be `accumulated <= amount`?

Reply:
I'll change the check order.

The function looks for unspent outputs until it has reached the required amount. If `<` would be changed to `<=` then it will keep looking for unspent outputs even thought it may have reached the required amount already.